### DISCUSSION TITLE
Windows Updates postponed... for one year?

### DISCUSSION THREAD

**Discussion by dantesquivel** [memstechtips/UnattendedWinstall#198] [Q&A]
Hi there - I'm somewhat new to this method of debloating Windows. I very much like it. I have 3 questions:

- I like the security updates, but the description says the features updates are merely delayed for 1 year?
- Can/should they be disabled indefinitely?
- If they can't be disabled, if had the experience of debloating Windows only to have an update install your standard bloatware, is that what will happen here after 1 year?

Thanks to anyone that my take the time to help.

**Comment by memstechtips** [comment]
Hey there and sorry for the late reply,

Since the creation of this issue I have released a new version of the XML file and a major update to [Winhance](https://winhance.net/), I recommend that you watch [this video](https://youtu.be/lrq3ph3xi50) which provides a full demo of the current version along with how to use the new version of WIMUtil and how to install the drivers etc. You now also have the ability to create an autounattend.xml file based on your selections in Winhance, so you have more control over which apps are removed and which settings are applied.

I have also created [this video](https://youtu.be/sCd1TI6AbOI) about v3.0.0 of UnattendedWinstall, I suggest that you watch it too if you're interested.

You can also completely disable the Windows Updates in Winhance.

I'm closing this issue now, but if you need more help with the same issue, feel free to reopen it.
